### PR TITLE
imported missing Poly function in trigsimp.py file 

### PR DESCRIPTION
**Pull request by arihantparsoya**
fix for #11062 
@asmeurer can you review this?

### CODE REVIEW

**Comment by aktech** [comment]
Please add tests.

**Comment by parsoyaarihant** [comment]
@aktech , I added the required tests.

**Comment by smichr** [comment]
+1

File: sympy/simplify/tests/test_trigsimp.py [hunk 1]
```diff
@@ -333,6 +334,7 @@ def test_trigsimp_groebner():
                              hints=[(tanh, x, y)]) == tanh(x + y)
 
 
+
```
Comment (aktech) [inline sympy/simplify/tests/test_trigsimp.py]: Why newline?

**Comment by parsoyaarihant** [comment]
@aktech , newline was a typo error and I fixed it.

**Comment by aktech** [comment]
Looks good. +1

**Pull request merged**